tests/test_env.py

```diff
@@ -153,3 +153,22 @@ def test_pip_needs_upgrade_mac_os_11(mocker, pip_version, arch):
         else:
             (uninstall_call,) = check_call.call_args_list
             assert uninstall_call[0][0][1:] == ['-m', 'pip', 'uninstall', 'setuptools', '-y']
+
+
+@pytest.mark.isolated
+@pytest.mark.skipif(IS_PY2, reason='venv module used on Python 3 only')
+@pytest.mark.skipif(IS_PYPY3 and os.name == 'nt', reason='Isolated tests not supported on PyPy3 + Windows')
```

Comment:
Added this until @gaborbernat can comment any with other ideas.

Reply:
So what's the issue here with pypy3 and windows? Didn't upstream fix this by now?

Reply:
I am pretty sure symlink in windows is not supported by PyPy. There are various issues open, like https://foss.heptapod.net/pypy/pypy/-/issues/3434 around that topic.